CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 # Changelog
 
+## 2021.3.0 (3 March 2021)
+
+### Enhancements
+
+1. Add ability to view a slice of the current variable in the data viewer using either axis/index dropdowns or a slice expression input field.
```

Comment:
I don't think we are enabling this in the March release, could we please remove this entry? Sorry for the trouble!